pkg/aggregator/aggregator.go

```diff
@@ -36,6 +37,22 @@ const bucketSize = 10                         // fixed for now
 // MetricSamplePoolBatchSize is the batch size of the metric sample pool.
 const MetricSamplePoolBatchSize = 32
 
+// hugeSeriesTagThreshold is the number of tags we consider "huge" and a sign
+// that something is probably wrong with the series.  If changing this value,
+// also change the description in tlmHugeSeries.
```

Comment:
```suggestion
// hugeSeriesTagThreshold is the number of tags we consider "huge" and a sign
// that something is probably wrong with the series. If changing this value,
// also change the description in tlmHugeSeries.
```

Reply:
We might have to agree to disagree on this, or add a lint for it.  When I learned to type, full-stops were followed by two spaces.  I'll change it in this PR, but I don't expect I can manage to avoid doing this every time I write a sentence.  Also [the two spaces were useful in a monospaced context](https://www.quora.com/In-the-past-periods-full-stops-were-followed-by-two-spaces-Lately-its-become-common-to-see-just-one-space-In-the-modern-world-should-we-use-two-spaces-between-sentences-or-just-one-is-okay-Does-it-depend-on-the?share=1) and this is a monospace context :)